browser/js/adminDash/orders/adminDash.orderForm.html

```diff
@@ -1,17 +1,180 @@
+                        </p>
+                    <button class="btn btn-raised btn-primary" ng-click="removeFromCart(song)" id="remove">Remove Song from Order</button>
+                  </div> -->
+                </div>
+              </td>
+              <td class="col-lg-2 col-md-2 col-sm-2 col-xs-2">
+                <p>{{song.price * song.song_order.quantity | currency}}</p>
+              </td>
+            </tr>
+            <tr>
+                <td class="col-lg-10 col-md-10 col-sm-10 col-xs-10" >
+                    <p id="total">Total:  </p>
+                </td>
+                <td class="col-lg-2 col-md-2 col-sm-2 col-xs-2">
+                    <p>{{order.total | currency}}</p>
+                </td>
+            </tr>
+        </table>
+        <!-- Implement if time <a ng-click="editOrder=!editOrder">Edit Order Information</a>-->
+      </div>
+  </div>
+  <!--Implement if time
+  <div ng-show="editAddress">
+    <header class="text-center">
+      <h3>Edit Shipping Address</h3>
+    </header>
+      <div class="well cart-container">
+        <form class="form-inline" role="form" novalidate name="addressForm" ng-submit="addressForm.$valid && updateAddress(orderAddress, order)">
+          <fieldset>
+            <div class="row">
+              <div class="form-group col-md-6">
+                <label for="firstName" class="control-label">First Name</label>
+                <div>
+                  <input class="form-control" name="firstName" type="text" ng-model="orderAddress.firstName" required>
+                </div>
+              </div>
+              <div class="form-group col-md-6">
+                <label for="lastName" class="control-label">Last Name</label>
+                <div>
+                  <input class="form-control" name="lastName" type="text" ng-model="orderAddress.lastName" required>
+                </div>
+              </div>
+            </div>
+            <div class="row">
+              <div class="form-group col-md-12">
+                <label for="address" class="control-label">Address</label>
+                <div>
+                  <input class="form-control" name="address" type="text" ng-model="orderAddress.address" required>
+                </div>
+              </div>
+            </div>
+            <div class="row">
+              <div class="form-group col-md-6">
+                <label for="city" class="control-label">City</label>
+                <div>
+                  <input class="form-control" name="city" type="text" ng-model="orderAddress.city" required>
+                </div>
+              </div>
+              <div class="form-group col-md-2">
+                <label for="state" class="control-label">State</label>
+                <div>
+                   <select class="selectpicker" name="state" ng-model="orderAddress.state" required>
+                      <option value="AK">AK</option>
```

Comment:
see browser/js/signup/signup.html line 40 on accounts branch for ng-repeat refactor